src/test/layout_test.cpp

```diff
@@ -48,8 +48,64 @@ TEST(Layout, BatchPositionInvalid) {
     EXPECT_EQ(Layout{"N.CH"}.getBatchIndex(), std::nullopt);
     EXPECT_EQ(Layout{"..NHW."}.getBatchIndex(), std::nullopt);
     EXPECT_EQ(Layout{"N...N"}.getBatchIndex(), std::nullopt);
-    EXPECT_EQ(Layout{"N...C...H"}.getBatchIndex(), std::nullopt);
+    EXPECT_EQ(Layout{"N...C...H"}.getBatchIndex(), std::nullopt);  // TODO validate as incorrect
```

Comment:
This is already validated as incorrect. `Layout{"N...C...H"}.validate()` should return false.